package/Dockerfile.submariner-libreswan-git

```diff
@@ -0,0 +1,41 @@
+FROM registry.fedoraproject.org/fedora:32
+
+RUN dnf -y install --nodocs git audit-libs-devel bison curl-devel fipscheck-devel flex \
+    gcc ldns-devel libcap-ng-devel libevent-devel \
+    libseccomp-devel libselinux-devel make nspr-devel nss-devel \
+    pam-devel pkgconfig systemd-devel unbound-devel xmlto rpm-build \
+    openldap-devel hostname nss-tools
+
+
+WORKDIR /root
+
+RUN git clone https://github.com/libreswan/libreswan
+RUN cd libreswan && git checkout main
+RUN cd libreswan && \
+    sed -i -e "s/^%global with_cavstests 1*/%global with_cavstests 0/g" packaging/fedora/libreswan.spec 
+
+RUN cd libreswan && make git-rpm 
+
+FROM registry.fedoraproject.org/fedora-minimal:32
```

Comment:
Please use the current engine `Dockerfile` — we switched away from the minimal Fedora image.

Reply:
Oh it seems like I was working on a an updated tree.